web/src/components/upload/UploadsTable.vue

```diff
@@ -0,0 +1,133 @@
+<script lang="ts" setup>
+import { computed, ref } from "vue";
+import { DateTime } from "luxon";
+import { useVModel } from "@vueuse/core";
+import { UploadReport } from "@/types/uploads/UploadReport";
+import { useReportsStore } from "@/stores";
+import UploadStatus from "./UploadStatus.vue";
+import UploadsTableInfoDialog from "./UploadsTableInfoDialog.vue";
+import UploadsTableDeleteDialog from "./UploadsTableDeleteDialog.vue";
+
+interface Props {
+  search?: string;
+}
+const props = withDefaults(defineProps<Props>(), {});
+const emit = defineEmits(["update:search", "update:reports"]);
+const reports = useReportsStore();
+
+// Table search value.
+const search = useVModel(props, "search", emit);
+
+// Table headers
+const headers = computed(() => [
+  { text: "Name", value: "name", sortable: true },
+  { text: "Upload date", value: "date", sortable: true },
+  { text: "Status", value: "status", sortable: true },
+  { text: "", value: "actions", sortable: false, align: "right" },
+]);
+
+// Table items
+// In the format for the Vuetify data-table.
+const items = computed(() =>
+  reports.reports.map((report) => ({
+    name: report.name,

```

Comment:
Maybe we should also show the local reference next to the report name, in case multiple datasets with the same name were uploaded?

Reply:
I am not sure about this yet, as the table will become too cramped. I was under the impression the date would be enough of an indicator to distinguish the different datasets?